branching_xblock/static/js/src/branching_xblock.js

```diff
@@ -1,6 +1,23 @@
 function BranchingXBlock(runtime, element) {
     const $el = $(element);
 
+    let currentHintNodeId = null;
+    let isHintVisible = false;
+
+    function setHintVisibility(visible) {
+        const $details = $el.find('[data-role="hint-collapsible"]');
+        if (!$details.length) {
+            isHintVisible = Boolean(visible);
+            return;
+        }
+        const $summary = $details.find('[data-role="hint-summary"]');
+        isHintVisible = Boolean(visible);
```

Comment:
```suggestion
        isHintVisible = Boolean(visible);
        if (!$details.length) {
            return;
        }
        const $summary = $details.find('[data-role="hint-summary"]');
```

Minor refactor to avoid duplication.